src/extensions/default/AutoUpdate/main.js

```diff
@@ -483,6 +483,9 @@ define(function (require, exports, module) {
 
             //Initiate the auto update, with update params
             initiateAutoUpdate(updateParams);
+	    
+	    //Send a truthy value to ensure caller is informed about successful initialization of auto-update
+	    return true;
```

Comment:
This seems okay to me. 

Just one nit, in UpdateNotification.js the _updateProcessHandler is actually an async function(internally calls initiateAutoUpdate), but  _defaultUpdateProcessHandler would work synchronously. The way the code is written seems to imply that the update was successfully handled, which is not the case even if the initialization was successful. Should we change the variable name to `initSuccess` in UpdateNotification.js?
On the otherhand, we can return a promise from here and resolve it with true or false accordingly to indicate actual success, but that would make things a bit complicated.

Reply:
Understood. Will change the variable name in that case. It can't be async, or rather we can't wait till success of auto-update process itself, because that would break the continuity of user journey in the workflow. No point in launching brackets.io after 5 minutes if download has failed.

Reply:
Oh yes, definitely. What I meant was to handle the initialization failure scenario separately from update success/failure scenario, if required but through a promise since that will be more reliable. As for update failure, we're already telling the user to go to the site if the update fails [here](https://github.com/adobe/brackets/blob/master/src/extensions/default/AutoUpdate/main.js#L247).

Reply:
Changed `success` to `initSuccess` now.